common/source/docs/common-advanced-battery-setup.rst

```diff
@@ -0,0 +1,38 @@
+.. _common-advanced-battery-setup:
+
+=============================================
+Advanced Advanced Power Monitor Configuration
+=============================================
+
+In addition to the normal battery current and voltage setup and scaling parameters addressed in each individual :ref:`power monitor's<common-powermodule-landingpage>` setup page, there are several other parameters available for advanced configuration. Each parameter is well documented by following its link below:
+
+.. note:: While links are provided for the first battery monitor parameter group, there can be up to 10 total monitors in the system depending on the individual autopilot's capabilities and/or use of SMBUS or UAVCAN monitors. These are accessed using ``BATTx_`` parameter prefixes
+
+Parameters for advanced power monitor failsafe setup
+----------------------------------------------------
+
+:ref:`BATT_WATT_MAX<BATT_WATT_MAX>`
```

Comment:
The BATT_WATT_MAX param is only available on Plane by the way not that it hurts having it too much.

Reply:
its listed in copter and rover params...is this a mistake?

Reply:
I think the script that produces the param wiki page can't handle the vehicle specific tags in the parameters.

> AP_GROUPINFO_FRAME("WATT_MAX", 8, AP_BattMonitor_Params, _watt_max, 0, AP_PARAM_FRAME_PLANE),

So in this example it's not reading that last AP_PARAM_FRAME_PLANE part

Reply:
Sounds like we need a mod to our param build script....will put in an issue....